src/main/java/org/mifos/connector/mojaloop/ilp/Ilp.java

```diff
@@ -0,0 +1,68 @@
+package org.mifos.connector.mojaloop.ilp;
+
+/*
+ * This Source Code Form is subject to the terms of the Mozilla
```

Comment:
not very sure about this license , better remove them , it can be added by tools like spotless later

Reply:
removed